ansible/roles_ocp_workloads/ocp4_workload_paloalto_prismacloud/.yamllint

```diff
@@ -0,0 +1,14 @@
+---
+extends: default
+
+rules:
+  comments:
+    require-starting-space: false
+  comments-indentation: disable
+  indentation:
+    level: warning
+    indent-sequences: consistent
+  line-length:
+    level: warning
+    max: 325
+    allow-non-breakable-inline-mappings: true
```

Comment:
The directory for `roles_ocp_workloads` already includes a `.yamllint`. No need to add one within the role directory as this will just result in redundant checks.

Reply:
@jkupferer removed .yamllint!